host_cleanup.sh

```diff
@@ -34,3 +34,5 @@ if [ "$?" == "0" ]; then
      sudo virsh net-destroy provisioning
      sudo virsh net-undefine provisioning
 fi
+
+sudo rm -rf /root/.vbmc
```

Comment:
Is it always /root or just $HOME? We support dev-scripts as non-root users too, right? Maybe `$HOME/.vmbc`?

Reply:
I run everything as a normal user and the configs are under /root/.

Reply:
I guess this should be in the teardown ansible playbook, but seems like a harmless enough quick fix :-)

Reply:
Note that I've already added this cleanup in the teardown playbook in PR #500 :)

Reply:
https://github.com/openshift-metal3/dev-scripts/pull/500/commits/63c0d0fccbc62132f16e5dac2bc04f4037cea999 FYI

Reply:
awesome, thanks @hardys .  we can just merge that one then